natgeo/mortar
# [SYTO-1192][SYTO-1193] Small css tweaks to cut down on unnecessary painting across components

To test:

Pull SYTO-1192+SYTO-1193

- npm test
- gulp serve
- Head over to http://localhost:3000/components/
- With dev tools open and paint flashing turned on, hover over story cards and POD promo
- Verify paint flashing isn't impacting entire page or neighboring components on hover
